app/assets/javascripts/app.js

```diff
@@ -17,7 +17,7 @@ document.addEventListener("DOMContentLoaded", function(event) {
             lead.showEvents = false;
           });
           this.leads = response;
-          this.leads = _.sortBy(this.leads, ['created_at']);
+          this.leads = _.orderBy(this.leads, ['most_recent_event'], ["desc"]);
```

Comment:
Here is the documentation for the `order_by` function https://lodash.com/docs/4.17.5#orderBy